### ISSUE TITLE
multimc fails to load after updationg to ubuntu 18.04.1 LTS

### ISSUE THREAD

**Issue by penguinrepair** [MultiMC/MultiMC5#2401]
System Information
-----------------------------
MultiMC version:

Operating System:

Summary of the issue or suggestion:
----------------------------------------------

What should happen:
------------------------------

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.

2.

3.

Suspected cause:
---------------------------

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

Additional Info:
---------------------------

**Comment by Janrupf** [comment]
We need more info

**Comment by UmbralVeritas** [comment]
I'm on Ubuntu 18.04.1 LTS (upgraded from 18.04) and I have no problems with MultiMC. 
I'm using [webupd8team's PPA for the latest Oracle Java 8](http://www.webupd8.org/2012/09/install-oracle-java-8-in-ubuntu-via-ppa.html), rather than the default java implementation. I don't know if that would affect whether or not MultiMC loads but it definitely helps Minecraft load.

Try backing up your instances, purging MultiMC, and then reinstalling it. If you still have a problem, you should fill out the blanks from template and be sure to include any error information from your logs. (Without details, you might as well have just posted "Computer broken".  If you're coming up short on information, penguinrepair, try running it from a terminal, Ctrl+Alt+T.)

**Comment by peterix** [comment]
This probably needs more details. 'fails to load' is so vague that it could mean anything.

Did:
* the game fail to load?
* the launcher failed to load?
* some random forge library now fails to load and the game ended up being vanilla?
* the screenshots fail to load?

Please be precise. And if you can't be precise, then at least use enough words to make whatever you write unambiguous.

Right after that, I'll probably ask you for logs.

**Comment by einare** [comment]
Not sure if I should comment since this old comments.
But Starting MultiMC5 for the first time gets you the files as it is installer.  But after that you see only the loading wheel for a while and then it stops.  So after a while trying to find crash or error report I ran the file in terminal and from the it prints out error.  it printed out this message: "/MultiMC: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory" 
Ok, so I went in /usr/lib and did not find the file there. I reinstalled libdouble-conversion1 but that did not do anything.
So since I am not a linux genuies  am not sure what else I can do.
